custom_connectors/custom_auth/trackvia_connector.rb

```diff
@@ -0,0 +1,712 @@
+      'all records</span>&nbsp;in a&nbsp;' \
+      "<span class='provider'>TrackVia view</span>.",
+      help: 'Delete all records in a TrackVia view',
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an application view from the list above'
+        }
+      ],
+      execute: lambda { |_connection, input|
+        delete("/openapi/views/#{input['view_id']}/records/all")
+          .after_error_response(/.*/) do |code, body, header, message|
+            error("#{message} : #{body}")
+          end
+      }
+    }
+  },
+  triggers: {
+    new_record: {
+      description: "Created&nbsp;<span class='provider'>" \
+      'record</span>&nbsp;in&nbsp;' \
+      "<span class='provider'>TrackVia</span>.",
+      help: 'Triggers whenever a record is created and&nbsp;' \
+      'is added to a specified TrackVia view.',
+      type: :paging_desc,
+      config_fields: [
+        {
+          name: 'app_name',
+          label: 'App',
+          type: 'string',
+          control_type: 'select',
+          pick_list: 'apps',
+          optional: false,
+          change_on_blur: true,
+          hint: 'Select a TrackVia application from the list above'
+        },
+        {
+          name: 'view_id',
+          label: 'View',
+          type: 'integer',
+          control_type: 'select',
+          pick_list: 'views',
+          pick_list_params: { app_name: 'app_name' },
+          optional: false,
+          hint: 'Select an application view from the list above'
+        }
+      ],
+      webhook_notification: lambda do |_input, payload|
+        # payload[0]
+        # HACK: This is a quick fix to replace an
+        # issue with webhook responses returning 'id'
+        # instead of 'ID'
+        hash = payload[0]
+        hash['ID'] = hash.delete 'id'
+        hash
+      end,
+      webhook_subscribe: lambda do |webhook_url, _connection, input, _recipe_id|
+        post("/openapi/zapier/views/#{input['view_id']}" \
+        '/api/hooks',
+             target_url: webhook_url,
+             event: 'created')
+      end,
+      webhook_unsubscribe: lambda do |webhook, input|
+        delete("/openapi/zapier/views/#{input['view_id']}" \
+          "/api/hooks/#{webhook['id']}")
+      end,
+      output_fields: lambda { |object_definitions|
+        object_definitions['hook_body']
+      }
+    },
+    updated_record: {
+      description: "Updated&nbsp;<span class='provider'>" \
```

Comment:
Remove the `&nbsp;` characters and use space. update the description and help as below:
Is the below description makes sense?
```
      description: "Updated <span class='provider'>" \
      'record</span> in ' \
      "<span class='provider'>TrackVia</span> view",
      help: 'Triggers whenever a record is updated in a TrackVia view ',
```

Reply:
Removed the `&nbsp;` characters. Consulting with product regarding description/help language.